File: Python/Warmup-1/monkey_trouble.py
# We have two monkeys, a and b, and the parameters a_smile and b_smile
# indicate if each is smiling. We are in trouble if they are both smiling
# or if neither of them is smiling. Return True if we are in trouble.
def monkey_trouble(a_smile, b_smile):
    return ( (a_smile and b_smile) or (not a_smile and not b_smile) )

print(monkey_trouble(True, True)) #true
print(monkey_trouble(False, False)) #true
print(monkey_trouble(True, False)) #false
print(monkey_trouble(False, True)) #false

File: Java/Warmup-1/MonkeyTrouble.java
/*
 * We have two monkeys, a and b, and the parameters aSmile and bSmile indicate
 * if each is smiling. We are in trouble if they are both smiling or if neither
 * of them is smiling. Return true if we are in trouble
 */
public class MonkeyTrouble {
	public static void main(String[] args) {
		System.out.println(monkeyTrouble(true, true)); //true
		System.out.println(monkeyTrouble(false, false)); //true
		System.out.println(monkeyTrouble(true, false)); //false
		System.out.println(monkeyTrouble(false, true)); //false
	}

	public static boolean monkeyTrouble(boolean aSmile, boolean bSmile) {
		return (aSmile && bSmile || (!aSmile && !bSmile));
	}
}
